### PR TITLE
New symbol for ParametersAction

### PR DESCRIPTION
**Pull request by mawinter69**
Replace the old style document-properties svg for the parameters action with the slightly
modified options icon from ionicons as symbol-parameters.
Add a translation for the document-properties to the parameters symbol.
A search in the sources of the jenkinsci org revealed only 3 other
plugins using the document-properties icon:
- jobrevision
- envinject (but deprecated)
- envinject-lib

Before: 
image

After: 
image

See [JENKINS-XXXXX](https://issues.jenkins.io/browse/JENKINS-XXXXX).

### Proposed changelog entries

* Entry 1: UI: New symbol for the Parameters action of builds

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6862"></a>

### CODE REVIEW

File: core/src/main/java/hudson/model/ParametersAction.java
Comment (daniel-beck) [inline core/src/main/java/hudson/model/ParametersAction.java:211]: 👍

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (daniel-beck) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:600]: Unsure about this one. There seems to be quite the semantic difference between the two.

Comment (NotMyFault) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:600]: This is no original tango svg, but a remixed one based on tango components. 

A more matching ionicon is the plain wrench, going off the tango base:
Screenshot 2022-07-16 at 17 38 05
But the proposed one looks fine for build parameters to me.

Comment (daniel-beck) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:600]: >  But the proposed one looks fine for build parameters to me.

Agree per https://github.com/jenkinsci/jenkins/pull/6862#discussion_r922680390, just this replacement doesn't look that useful.

**Review by daniel-beck — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
